tobac/tracking.py

```diff
@@ -24,31 +45,124 @@ def linking_trackpy(
+        jeopardising the tracking procedure. To prevent this, tobac uses
+        an additional minimum radius of the search range. [5]_
+        Default is None.
+
+    subnetwork_size : int, optional
+        Maximum size of subnetwork for linking. This parameter should be
+        adjusted when using adaptive search. Usually a lower value is desired
+        in that case. For a more in depth explanation have look
+        `here <https://soft-matter.github.io/trackpy/v0.3.0/tutorial/adaptive-search.html>`_
+        If None, 30 is used for regular search and 15 for adaptive search.
+        Default is None.
+
+    v_max : float, optional
+        Speed at which features are allowed to move. Default is None.
+
+    memory : int, optional
+        Number of output timesteps features allowed to vanish for to
+        be still considered tracked. Default is 0.
+        .. warning :: This parameter should be used with caution, as it
+                     can lead to erroneous trajectory linking,
+                     espacially for data with low time resolution. [5]_
+
+    stubs : int, optional
+        Minimum number of timesteps of a tracked cell to be reported
+        Default is 1
+
+    time_cell_min : float, optional
+        Minimum length in time of tracked cell to be reported in minutes
+        Default is None.
+
+    order : int, optional
+        Order of polynomial used to extrapolate trajectory into gaps and
+        ond start and end point.
+        Default is 1.
+
+    extrapolate : int, optional
+        Number or timesteps to extrapolate trajectories.
+        Default is 0.
+
+    method_linking : {'random', 'predict'}, optional
+        Flag choosing method used for trajectory linking.
+        Default is 'random'.
+
+    adaptive_step : float, optional
+        Reduce search range by multiplying it by this factor.
+
+    adaptive_stop : float, optional
+        If not None, when encountering an oversize subnet, retry by progressively
+        reducing search_range until the subnet is solvable. If search_range
+        becomes <= adaptive_stop, give up and raise a SubnetOversizeException.
+        Default is None
+
+    cell_number_start : int, optional
+        Cell number for first tracked cell.
+        Default is 1
+
     cell_number_unassigned: int
-        Number to set the unassigned/non-tracked cells to. By default, this is -1.
-        Note that if you set this to `np.nan`, the data type of 'cell' will
-        change to float.
+        Number to set the unassigned/non-tracked cells to. Note that if you set this
+        to `np.nan`, the data type of 'cell' will change to float.
+        Default is -1
+
+    Returns
+    -------
+    trajectories_final : xarray.Dataset
+        This enables filtering the resulting trajectories, e.g. to
+        reject trajectories that are only partially captured at the
+        boundaries of the input field both in space and time. [5]_
```

Comment:
Where is this reference to?

Reply:
I don't know, this is some fragment from older parts of the documentation I overlooked. I removed it and also changed this part for the tracking output to actually explain the resulting object.